src/debug.cpp

```diff
@@ -342,31 +342,34 @@ static void debug_error_prompt(
     };
     init_window( ui );
     ui.on_screen_resize( init_window );
-    const std::string message = string_format(
-                                    "\n\n" // Looks nicer with some space
-                                    " %s\n" // translated user string: error notification
-                                    " -----------------------------------------------------------\n"
-                                    "%s"
-                                    " -----------------------------------------------------------\n"
+    const std::string error_message = string_format(
+                                          "\n\n" // Looks nicer with some space
+                                          " %s\n" // translated user string: error notification
+                                          " -----------------------------------------------------------\n"
+                                          "%s"
+                                          " -----------------------------------------------------------\n"
 #if defined(BACKTRACE)
-                                    " %s\n" // translated user string: where to find backtrace
+                                          " %s\n" // translated user string: where to find backtrace
 #endif
-                                    " %s\n" // translated user string: space to continue
-                                    " %s\n" // translated user string: ignore key
-#if defined(TILES)
-                                    " %s\n" // translated user string: copy
-#endif // TILES
-                                    , _( "An error has occurred!  Written below is the error report:" ),
-                                    formatted_report,
+                                          , _( "An error has occurred!  Written below is the error report:" ),
+                                          formatted_report,
 #if defined(BACKTRACE)
-                                    backtrace_instructions,
+                                          backtrace_instructions
 #endif
-                                    _( "Press <color_white>space bar</color> to continue the game." ),
```

Comment:
This is weird. This doesn't build locally because BACKTRACE is not defined in local builds. It happens to be set in CI because it seems like we just set it by default everywhere. The trailing comma on `formatted_report,` cause a syntax error because there is no argument after if BACKTRACE is not defined.

Reply:
Ah #81073 addresses this.